lib/plenario2_web/templates/layout/app.html.eex

```diff
@@ -9,6 +9,23 @@
 
     <title>Plenario</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <style>
```

Comment:
i couldn't figure out how to effect this via `app.css` so i rammed it in here.

basically, i hate being forced into a 768px view port on a modern laptop screen. it just makes everything compressed and look like shit.